svg/pause.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024"><path d="M409.6 384h76.8v256h-76.8V384zm128 0v256h76.8V384h-76.8z"/></svg>
```

Comment:
The order is not significant. The order change is probably because Sketch puts these properties in a different order to adobe illustrator.